website/src/data/users.js

```diff
@@ -998,6 +998,14 @@ const Users = [
     source: null,
     tags: [ 'facebook'],
   },
+  {
+    title: 'Tinaël Devresse',
+    description: 'Personal website of Tinaël Devresse, a blog about anything that cross my mind which I think could interest y\'all.',
+    preview: require('./showcase/tinaeldevresse.png'),
+    website: 'https://tinaeldevress.eu',
```

Comment:
```suggestion
    website: 'https://tinaeldevresse.eu/',
```

hopefully I double check everything 😅